### ISSUE TITLE
Add dartfmt support

### ISSUE THREAD

**Issue by klavs** [dint-dev/gpm#6]
Having `gpm format` or `gpm fmt` would be a nice addition for completeness.

https://flutter.dev/docs/development/tools/formatting#automatically-formatting-code-with-the-flutter-command

**Comment by terrier989** [comment]
I agree, but the challenge is that if two developers have two different versions of `dartfmt`, they may reformat the whole project differently in each commit, which would be unacceptable noise for code reviewers.

I guess a way to solve this would be some heuristics or configuration file that determines the dartfmt version committers should use.

**Comment by klavs** [comment]
> the challenge is that if two developers have two different versions of `dartfmt`, they may reformat the whole project differently in each commit, which would be unacceptable noise for code reviewers.
Isn't that a problem outside of scope of a package management tool?
I mean, having `gpm` run it on all packages does not introduce anything else than what you have running it without `gpm`.
